lib/spec_factory.hpp

```diff
@@ -41,8 +41,8 @@ class SpecFactory {
     void pick_neighbors(TimedSpec& spec, vector<TimedSpec>& neighbors);
 
     //**** WlSpec ****//
-    wl_spec_t random_wl_spec();
-    void pick_neighbors(wl_spec_t& spec, vector<wl_spec_t>& neighbors);
+    WlSpec* random_wl_spec();
+    void pick_neighbors(WlSpec*& spec, std::vector<WlSpec*>& neighbors);
```

Comment:
```suggestion
    void pick_neighbors(WlSpec* spec, std::vector<WlSpec*>& neighbors);
```

Reply:
[SpecFactory should use pointers, not references to pointers](https://github.com/all-things-networking/fperf/pull/27/commits/47322b25c8da96504aa178ade29c1898eed91129)